maps/DeadSpace/ishimura.dm

```diff
@@ -6,12 +6,13 @@
 	name = "Ishimura"
 	full_name = "USG Ishimura"
 	path = "ishimura"
-	station_levels = list(1,2,3) // Add 4 to this once the EVA Deck is complete, and remove this comment.
-	contact_levels = list(1,2) // Add 3 to this once the EVA Deck is complete, and remove this comment.
+	station_levels = list(1,2) // Add 3 to this once the EVA Deck is complete, and remove this comment.
+	contact_levels = list(1,2,3,4) // Add 4 to this once the EVA Deck is complete, and remove this comment.
```

Comment:
this line fixes comms and solnet, right?

Reply:
Correct.